src/session_impl.cpp

```diff
@@ -182,7 +182,7 @@ using namespace std::placeholders;
 #ifdef BOOST_NO_EXCEPTIONS
 namespace boost {
 
-	void throw_exception(std::exception const& e) { std::abort(); }
+	void throw_exception(std::exception const& e) { exit(0); }
```

Comment:
why this change?

Reply:
seems some systems have undefined behavior with abort....

Reply:
how so?

Reply:
i found this in sonarcloud, one of the code checkers you have, the description provided is:
```<stdlib.h>'s abort, exit, getenv, and system have undefined, and implementation-defined behaviors, and should therefore be used.```

 if you think is not neccessary i can remove it, but from atol and atoi i can confirm is deprecated.

Reply:
I find it hard to believe that `abort` has *undefined* behavior. It has *implementation* defined behavior, specifically the behavior that's intended here

Reply:
abort (on most normal computers) will raise a `SIGABRT` which will be caught by debuggers. That's the intended behavior. Not all systems have signals, but it is likely to do something else that also will be caught by debuggers